gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -76,6 +76,8 @@
  */
 public abstract class JdbcExtractor extends QueryBasedExtractor<JsonArray, JsonElement>
     implements SourceSpecificLayer<JsonArray, JsonElement>, JdbcSpecificLayer {
+  public static final String JOIN_OPERATION_REGEX = "([a-zA-Z_].*)\\.([a-zA-Z_].*)(\\s*)=(\\s*)([a-zA-Z_].*)\\.([a-zA-Z_].*)";
```

Comment:
Identifier names can have numbers. Is the problem the '=' in the where or is it the multiple tables in the from? There are also other join predicates, such as !=.
Would a more appropriate check be for multiple tables in the FROM list?